i18n-country-translations.gemspec

```diff
@@ -19,9 +19,9 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
   s.platform     = Gem::Platform::RUBY
 
-  s.add_dependency('i18n', '~> 1.0.1')
-  s.add_runtime_dependency "railties", '>= 4.0'
-  s.add_development_dependency "rails", "~> 5.2.0"
+  s.add_dependency('i18n', '>= 0.9.3', '< 2')
+  s.add_runtime_dependency 'railties', '>= 4.0'
+  s.add_development_dependency "rails", '>= 4.0'
```

Comment:
Any reasons we are setting this to `4.0`?

Reply:
Because I'm an idiot. I made a stupid copy paste.